cookbooks/aws-parallelcluster-install/attributes/default.rb

```diff
@@ -4,3 +4,5 @@
 default['cluster']['sysctl']['ipv4']['gc_thresh1'] = 0
 default['cluster']['sysctl']['ipv4']['gc_thresh2'] = 15_360
 default['cluster']['sysctl']['ipv4']['gc_thresh3'] = 16_384
+# Created this new variable to be able to mock it freely
+default['cluster']['platform_version'] = node['platform_version']
```

Comment:
Question: is it intended to ne in the install cookbook? Shouldn't be in the common cookbook since it is used there?

Reply:
(I do not get how it worked with the file in the wrong folder)